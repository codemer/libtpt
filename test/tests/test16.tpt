@set(str, "Some string here")\
@substr(str, 5, 6)
@substr(str, 0, 4)
@substr(str, 5)
@set(str, "Field 1\tField\\2")\
@concat("Sample\t", str)
