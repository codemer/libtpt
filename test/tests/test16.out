string
Some
string here
Sample	Field 1	Field\2
